test/README.md

```diff
@@ -182,18 +184,18 @@ call methods that interact with the bitcoind nodes-under-test.
 If further introspection of the bitcoind instances themselves becomes
 necessary, this can be accomplished by first setting a pdb breakpoint
 at an appropriate location, running the test to that point, then using
-`gdb` to attach to the process and debug.
+`gdb` (or `lldb` on macOS) to attach to the process and debug.
 
-For instance, to attach to `self.node[1]` during a run:
+For instance, to attach to `self.node[1]` during a run you can get
+the pid of the node within `pdb`.
 
-```bash
-2017-06-27 14:13:56.686000 TestFramework (INFO): Initializing test directory /tmp/user/1000/testo9vsdjo3
+```
+(pdb) self.node[1].process.pid
 ```
 
-use the directory path to get the pid from the pid file:
+Then you can use the pid to start `gdb`:
 
 ```bash
-cat /tmp/user/1000/testo9vsdjo3/node1/regtest/bitcoind.pid
```

Comment:
Why remove? Is this way no longer possible?

Reply:
It still works but getting the pid from within pdb is just much simpler. But I can add it back if we want to keep it in.

Reply:
Not everyone is running with pdb, so to keep the fallback would be nice

Reply:
Hm, just curious: is there another practical way to pause a functional test besides pdb? Anyway, since it doesn't hurt, I added it back now :)

Reply:
I rarely use pdb but often add some manual `input("Pause")` breakpoints. (while running the test directly, not via `test_runner`).

Reply:
@mzumsande I see. Thanks!

Reply:
```py
import time; time.sleep(10000);